<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="passportdata">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Passport information:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="seriaedit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="namedit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="mail">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;E-mail:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="year">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1919</number>
   </property>
   <property name="maximum">
    <number>2019</number>
   </property>
   <property name="value">
    <number>2019</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="do_show">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>221</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Show personal information in members list.</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="phone_edit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>260</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>+7</string>
   </property>
  </widget>
  <widget class="QLabel" name="seria">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Series:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="surname">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Surname:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passport_numedit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="pasport_num">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;â„–&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="phone">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Phone number:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="seria_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Place of registration:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Name:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="month">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>January</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>February</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>March</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>April</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>May</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>June</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>July</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>August</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>September</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>October</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>November</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>December</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="day">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>31</number>
   </property>
  </widget>
  <widget class="QLabel" name="birthday">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Date of birth:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mail_edit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="sex">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Sex:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addmember">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>350</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add member</string>
   </property>
  </widget>
  <widget class="QComboBox" name="sex_select">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>Male</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Female</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="surnamedit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
